tmk_core/protocol/chibios/usb_main.c

```diff
@@ -1051,45 +1052,44 @@ void virtser_task(void) {
 #ifdef JOYSTICK_ENABLE
 
 void send_joystick_packet(joystick_t *joystick) {
-    joystick_report_t rep = {
+    static joystick_report_t rep;
+    rep = (joystick_report_t) {
 #    if JOYSTICK_AXES_COUNT > 0
         .axes =
-            {
-                joystick->axes[0],
+        { joystick->axes[0],
```

Comment:
Did you run clang-format on this file? If you've made any style changes it doesn't like the bot will just open a PR to revert them.

Reply:
The code formatting changes are result of clang-format after the static keyword changes.